### PR TITLE
LG-4193: Inline links

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-4193](https://jira.mongodb.org/browse/LG-4193)

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

File: packages/typography/src/H1/H1.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    generate: {
+      combineArgs: {
+        darkMode: [false, true],
+        baseFontSize: [13, 16],
+      },
+      decorator: (Instance, context) => {
+        return (
+          <LeafygreenProvider
+            darkMode={context?.args.darkMode}
+            baseFontSize={context?.args.baseFontSize}
```
Comment (shaneeza) [inline packages/typography/src/H1/H1.stories.tsx:27]: Doesn't need `baseFontSize`. Same for the other headings

File: tools/storybook-utils/src/storybookArgTypes.ts [hunk 2]
```diff
@@ -70,6 +70,7 @@ export const storybookExcludedArgTypes: Record<string, StoryArgType> = {
   'aria-describedby': { control: 'none' },
   'aria-label': { control: 'none' },
   'aria-labelledby': { control: 'none' },
+  'data-lgid': { control: 'none' },
```
Comment (shaneeza) [inline tools/storybook-utils/src/storybookArgTypes.ts:73]: i don't think this is working

File: packages/typography/src/Body/Body.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { type StoryMetaType } from '@lg-tools/storybook-utils';
+
+import Body from './Body';
+import { BodyProps } from './Body.types';
+
+export const LiveExample = ({ children, ...rest }: BodyProps) => {
```
[the remark this answers is not in the document]
Comment (shaneeza) [reply packages/typography/src/Body/Body.stories.tsx:7]: that makes sense, i'll change the names and test on .design.

File: packages/typography/src/Body/Body.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+const meta: StoryMetaType<typeof Body> = {
+  title: 'Components/Typography/Body',
+  component: Body,
+  parameters: {
+    default: 'Basic',
+    generate: {
+      combineArgs: {
+        darkMode: [false, true],
+        baseFontSize: [13, 16],
```
Comment (stephl3) [inline packages/typography/src/Body/Body.stories.tsx:19]: can we update these numbers and others in subsequent stories to use `BaseFontSize`?

File: packages/typography/src/Body/Body.stories.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (shaneeza) [reply packages/typography/src/Body/Body.stories.tsx]: it looks the same on .design

**Review by stephl3 — APPROVED** [review]
LGTM

File: tools/storybook-utils/src/storybookArgTypes.ts (hunk 2 above)
Comment (shaneeza) [reply tools/storybook-utils/src/storybookArgTypes.ts:73]: add changeset or remove

**Pull request merged**